src/assets/svg/Linkedin.jsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+
+const Linkedin = ({ height, fill }) => (
+  <svg
+    height={height}
+    width={height}
+    fill={fill}
+    viewBox="0 0 144 144"
```

Comment:
can you change all the quotes here to singles? surprised lint didn't catch it. or maybe it's not checking these files.